### ISSUE TITLE
Inline-style image links with long title text violate rule MD013

### ISSUE THREAD

**Issue by robyaw** [DavidAnson/markdownlint#248]
When using [inline-style image links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images), if the combined number of characters in the alt text, the image link and title text is longer than the [`line-length` value](https://github.com/updownpress/markdown-lint/blob/master/rules/013-line-length.md), it results in a violation of [rule MD013 (line length)](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md013---line-length).

This can be partially mitigated by putting the quoted title text on a separate line, but this breaks highlighting within Visual Studio Code.

Note that the line-length rule is not violated if using reference-style image links where the combination of image link and title text would cause the line length to be longer than the `line-length` value.
